ansible/roles/swap/tasks/main.yml

```diff
@@ -20,7 +20,7 @@
 - name: Set swap file mode
   become: true
   file: path=/swapfile state=file mode=600
-  when: swap_size is defined and not swapfile.stat.exists
+  when: swap_size is defined and swapfile.stat.exists
```

Comment:
I think this `Set swap file mode` task should be moved down below the `Make swap` task. Also, I don't think this change is correct. Instead we should change this to simply `when: swap_size is defined`